Applications/Spire/Source/TimeAndSales/TimeAndSalesWindow.cpp

```diff
@@ -164,7 +164,9 @@ void TimeAndSalesWindow::export_table() {
 
 void TimeAndSalesWindow::show_properties_dialog() {
   TimeAndSalesPropertiesDialog dialog(m_properties, this);
-  dialog.connect_apply_signal([=] (auto p) { set_properties(p); });
+  dialog.connect_apply_signal([=, &dialog] {
```

Comment:
Similar principle applies, also fix indentation.

Reply:
Do we not double indent when breaking up lambdas into multiple lines?

Reply:
Nope.